src/server/event_bus.c

```
@@ -0,0 +1,168 @@
+
+/* Apply all pending removals accumulated during a fire(). */
+static void flush_removals(obc_event_entry_t *e)
+{
+    for (int r = 0; r < e->remove_count; r++)
+        remove_sub(e, e->remove_pending[r]);
+    e->remove_count = 0;
+}
+
+/* --- Public API ----------------------------------------------------------- */
+
+void obc_event_bus_init(void)
```

Cppcheck (2.19.0)

[style] The function 'obc_event_bus_init' is never used.

(unusedFunction)